### PR TITLE
Update deployment/build system instructions

### PR DESCRIPTION
**Pull request by florian-h05**
Signed-off-by: Florian Hotze <florianh_dev@icloud.com>

### CODE REVIEW

**Comment by florian-h05** [comment]
@digitaldan Gentle ping.
FYI: I am currently working on another PR that improves the type defs for the `actions` namespace.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -166,9 +141,61 @@ On the command line you can use `git commit -s` to sign off the commit.
...
+openhab-js is using NodeJS version 12.16.1 as the JS Scripting add-on is currently on this version.
+It is required to use this version as well in your development environment, otherwise you could cause trouble.
+
+Maintainers recommend to use a NodeJS version manager, see [nvm-sh/nvm](https://github.com/nvm-sh/nvm).
+
+openhab-js has several (development) dependencies which are required.
+
+Run `npm install` to install those dependencies.
+
+A few of those dependencies are required to be installed globally:
```
Comment (digitaldan) [inline CONTRIBUTING.md:160]: Is this really the case?  I think these are installed as dev dependencies, so no globals are required.  At least for me, i can run everything in package.json without globals.

**Review by digitaldan — APPROVED** [review]
Have one comment, but other then that LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (florian-h05) [reply CONTRIBUTING.md:160]: No it is not, thanks for the question.
I just rechecked with a new Node installation, I will correct this part.

I am wondering what made my think that they are required as globals.

**Comment by florian-h05** [comment]
@digitaldan 
I will merge this as you approved and your question has been resolved!

**Comment by digitaldan** [comment]
Lgtm!

**Pull request merged**